src/api/seeders/20190625074158-Article.js

```diff
@@ -0,0 +1,80 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('Articles',
```

Comment:
@Ramadhan0 I don't understand why you need this.

Reply:
I need to have data in  Articles table, this is because for a user to be an Author he/she needs to have atleast one article

Reply:
@Ramadhan0 rebase with develop to get latest changes